lib/logger-helper.js

```diff
@@ -152,13 +157,18 @@ function getResponseAudit(req, res, options) {
 
     headers = handleJson(headers, options.logger, options.response.excludeHeaders, options.response.maskHeaders);
 
+    let bodyStr = _.isEmpty(responseBody) ? NA: JSON.stringify(responseBody);
```

Comment:
the logic is the same both in request & response , I suggest to extract it to a common method, and move it to utils.js file.